mats/6.ms

```diff
@@ -2685,7 +2685,7 @@
       (begin
         (system "ln -s ../examples .")
         (load "examples/fatfib.ss" compile)
-        (system "/bin/rm examples")
+        (system "rm examples")
```

Comment:
This may need a `-f`.  Part of the reason for explicitly calling out `/bin/rm` is to avoid aliases of `rm` that change the defaults to prompt before removing files.

Reply:
Would just `(delete-file "examples")` work here?

Reply:
I think I'd jut add the `-f` in keeping consistent with the changes of `/bin/rm` to `rm` elsewhere.